addons/website/controllers/form.py

```diff
@@ -133,10 +133,14 @@ def many2many(self, field_label, field_input, *args):
         'monetary': floating,
     }
 
+    def _get_email_field_name(self, model_name):
```

Comment:
Are you looking to ``def _mail_get_primary_email_field()`` method maybe ?

Reply:
Hello @tde-banana-odoo ,
The function _mail_get_primary_email_field() functions correctly for the helpdesk ticket module. However, in other modules, it returns None instead of the expected 'email_from' field.

Thank You;)